it.baeyens.arduino.core/src/it/baeyens/arduino/managers/Manager.java

```diff
@@ -647,12 +648,12 @@ private static void link(File something, File somewhere) throws IOException, Int
     private static void chmod(File file, int mode) throws IOException, InterruptedException {
 	String octal = Integer.toOctalString(mode);
 	if (Platform.getOS().equals(Platform.OS_WIN32)) {
-	    boolean ownerExecute = (((mode / (8 * 8)) & 1) == 1);
-	    boolean ownerRead = (((mode / (8 * 8)) & 4) == 4);
-	    boolean ownerWrite = (((mode / (8 * 8)) & 2) == 2);
-	    boolean everyoneExecute = (((mode / 8) & 1) == 1);
-	    boolean everyoneRead = (((mode / 8) & 4) == 4);
-	    boolean everyoneWrite = (((mode / 8) & 2) == 2);
+	    boolean ownerExecute = ((mode / (8 * 8)) & 1) == 1;
+	    boolean ownerRead = ((mode / (8 * 8)) & 4) == 4;
+	    boolean ownerWrite = ((mode / (8 * 8)) & 2) == 2;
+	    boolean everyoneExecute = ((mode / 8) & 1) == 1;
+	    boolean everyoneRead = ((mode / 8) & 4) == 4;
+	    boolean everyoneWrite = ((mode / 8) & 2) == 2;
```

Comment:
Personally I prefer the original. I know the parenthesis is not needed but for me it separates "the calculation" from "the assignment". I have had hard to trace bug in the past due to missing "unnecessary brackets".
Anyone else has an opinion on this?